packages/teleport/src/Kubes/Kubes.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { Box, Indicator, ButtonPrimary } from 'design';
+import { Danger } from 'design/Alert';
+import KubeList from 'teleport/Kubes/KubeList';
+import {
+  FeatureBox,
+  FeatureHeader,
+  FeatureHeaderTitle,
+} from 'teleport/components/Layout';
+import useTeleport from 'teleport/useTeleport';
+import useKubes, { State } from './useKubes';
+
+export default function Container() {
+  const ctx = useTeleport();
+  const state = useKubes(ctx);
+  return <Kubes {...state} />;
+}
+
+const docUrl = 'https://goteleport.com/docs/kubernetes-access';
```

Comment:
string/numeric constants, defined on the global scope, need to be capitalized  `const DOC_URL = `